source/instructors/setupcourses/assignment-settings/mandatory.rst

```diff
@@ -0,0 +1,10 @@
+.. meta::
+   :description: Setting assignment to be mandatory or optional to be included or excluded from progress bar calculation
+
+
+.. _mandatory-assignments:
+
+Mandatory Progress
+==================
+
+By default all assignments are mandatory but can be made optional by toggling the switch off and will not then be included within the :ref:`progress bar <progress-bar>` calculation for students.
```

Comment:
By default all assignments are mandatory but can be made optional by toggling the switch off. If Mandatory is toggled off the assignment  will not be included within the :ref:`progress bar <progress-bar>` calculation for students.

Reply:
the above is line 10